pkg/statsd/statsd_test.go

```diff
@@ -2,8 +2,10 @@ package statsd
 
 import (
 	"context"
+	"github.com/stretchr/testify/assert"
```

Comment:
There should be a check somewhere for this, not sure why the build didn't hit it.  Import ordering is:

- stdlib
- 3rd party
- local